skeleton/nix/stack-shell.nix

```diff
@@ -0,0 +1,23 @@
+# This is the derivation used by "stack --nix".
+# It provides the system dependencies required for a stack build.
+{ iohkSkeletonPackages ? import ./.. {}
```

Comment:
To make it easier to merge in upstream changes to these files, I think it's a good idea to avoid using the project name unless we have to. I'd call this something like `localPackages`.

Reply:
:+1: